lib/model/message_list.dart

```diff
@@ -86,6 +87,30 @@ class MessageListView extends ChangeNotifier {
     notifyListeners();
   }
 
+  Message applyChangesToMessage(UpdateMessageEvent event, Message oldMessage) {
```

Comment:
Thoughts on where this function should live? It feels a little out of place here in `message_list`. I was thinking it might be a little better in `events.dart`, so that I don't need to import `events.dart` here. Thoughts?

Reply:
Ok, resolving!